std_msgs/String class_name
float64[] position
time stamp
---
bool success